docs-src/concepts/what-is-an-update.md

```diff
@@ -0,0 +1,43 @@
+---
+id: what-is-an-update
+title: What is an Update?
+sidebar_label: Update
+description: An Update is a request to and a response from Workflow Execution.
+tags:
+  - term
+  - updates
+  - explanation
+---
+
+An Update is a request to and a response from a Temporal Client to a [Workflow Execution](/concepts/what-is-a-workflow-execution).
+
+- [How to develop, send, and handle Updates in Go](/go/updates)
+
+The Workflow must have a function to handle the Update.
+Unlike a [Signal](/concepts/what-is-a-signal) handler, the Update handler function can mutate the state of the Workflow while also returning a value to the caller.
+The Update handler listens for Updates by the Update's name.
+
+When there is the potential for multiple Updates to cause a duplication problem, Temporal recommends adding idempotency logic to your Update handler that checks for duplicates.
+
+An Update has four phases.
```

Comment:
I think having "phases" separate from "states" is a bit confusing, but I guess not that bad. May want to list states somewhere too.

Reply:
The thinking here was that states are the endpoints of phases of execution. i.e. the ACCEPTED state is the completion state of the validation phase. That's why the completion states are called out below.